backend/LexBoxApi/Jobs/LexJob.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Diagnostics;
+using LexBoxApi.Otel;
+using OpenTelemetry.Trace;
+using Quartz;
+
+namespace LexBoxApi.Jobs;
+
+public abstract class LexJob : IJob
+{
+    public string? TraceParentId { get; set; }
```

Comment:
Something like that?
```suggestion
    public string? JobTriggerTraceId { get; set; }
```